src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Globalization;
```

Comment:
not sure why this is needed.  but don't you need a `using` for `Microsoft.AspNet.Razor.Runtime`?